src/benchmark/file_io_read_mmap_benchmark.cpp

```diff
@@ -1,19 +1,24 @@
 #include "file_io_read_micro_benchmark.hpp"
 
+#ifdef __linux__
 #include <umap/umap.h>
+#endif
```

Comment:
This has changes of Tobias Umap fix included, as I wanted to develop on Mac. As soon as his PR is merged this should no loger be a change